### DISCUSSION TITLE
Extract the SQL error message from SQLAlchemy error

### DISCUSSION THREAD

**Discussion by avin-kavish** [sqlalchemy/sqlalchemy#10035] [Usage Questions]
When a query errors due to a problem with the SQL, the SQLAlchemy error shows a very verbose error message that includes the original SQL, links to more details etc.

I just want to get the error message returned from the database server. For example - `column 'name' doesn't exist in relation 'user'`. How do I get this?

**Comment by CaselIT** [comment]
Hi.

In general you can use `.orig` to access the original exception. Documented as https://docs.sqlalchemy.org/en/20/core/exceptions.html#sqlalchemy.exc.StatementError.orig
